tests/civic-cookie-control.spec.ts

```diff
@@ -1,5 +1,9 @@
 import generateCMPTests from '../playwright/runner';
 
-generateCMPTests('civic-cookie-control', ['https://www.birmingham.gov.uk/', 'https://planning.org.uk/', 'https://www.jessops.com/'], {
-    skipRegions: ['US', 'DE'],
-});
+generateCMPTests(
+    'civic-cookie-control',
+    ['https://www.birmingham.gov.uk/', 'https://planning.org.uk/', 'https://www.jessops.com/', 'https://www.bcs.org/'],
```

Comment:
I tested against these URLs and noticed https://planning.org.uk/ had an unhandled CPM. Looks like they’re no longer using the civic-cookie-control CMP. We could remove it from this list.